polynote-frontend/style/styles.less

```diff
@@ -1804,15 +1804,24 @@ button {
   &.right-collapsed {
     grid-template-columns: max-content 1px auto 1px 2em;
 
-    .kernel-ui h2 {
-      transform: translateY(-100%) rotate(90deg);
-      transform-origin: bottom left;
-      width: fit-content;
-      &:hover {
-        background-color: @ui-background-accent;
+    .right .ui-panel {
```

Comment:
Could probably extract this, since it's the same for both `&.right-collapsed .right` and `&.left-collapsed .left`. But I'm not sure if that's more confusing